mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/RabbitMQEventBus.java

```diff
@@ -143,6 +145,9 @@ public Registration register(MailboxListener listener, Group group) {
     public Mono<Void> reDeliver(Group group, Event event) {
         Preconditions.checkState(isRunning, NOT_RUNNING_ERROR_MESSAGE);
         if (!event.isNoop()) {
+            if (group instanceof EventDispatcher.DispatchingFailureGroup) {
```

Comment:
I can't understand that line, could you explain?

Reply:
if the eventBus.dispatch() gets error while dispatching an event (rabbitMQ network outage maybe), which means all the group consumers will not be receiving that event.

I store the that event in the dead letter and expecting in the future, it will be dispatched again not only for a specific consumer but all.

That's why it is special, and I need to check event type before processing further.

Reply:
Would you mind writing a comment about that?

Reply:
I will be adding it.